samples/sample_calibrate_eye_to_hand.py

```diff
@@ -75,6 +77,14 @@ def _main():
     else:
         print("FAILED")
 
+    print("Transforming calibration result")
+    transformed_numpy_matrix = calibration_result.transform()
+
+    print("Saving transformed calibration result")
```

Comment:
Same here, update comment, to fex "Saving calibration result transformation matrix" or similar.

Reply:
Updated it to say `"Saving calibration result transformation matrix"`.